commit 1ad282c7141bdb6fe13a2cd03264dc9e77997226 (refs/changes/52/3852/2)
Author: Nick Zivkovic <nick.zivkovic@joyent.com>
Date:   2018-04-20T22:21:23+00:00 (1 year, 6 months ago)
    
    TRITON-327 sdcadm gets VMAPI-error on fabric-setup
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Trent Mick <trentm@joyent.com>

diff --git a/lib/post-setup/fabrics.js b/lib/post-setup/fabrics.js
index d08c3f7..24745c7 100644
--- a/lib/post-setup/fabrics.js
+++ b/lib/post-setup/fabrics.js
@@ -712,7 +712,7 @@ Fabrics.prototype.updateFabricsSvcs = function updateFabricsSvcs(cb) {
     if (!self.configChanged) {
         return cb(null);
     }
-    var svcs = ['napi', 'vmapi', 'dhcpd'];
+    var svcs = ['napi', 'dhcpd', 'vmapi'];
 
     self.progress('Restarting config of services using "fabric_cfg": %s',
     svcs.join(', '));
@@ -720,7 +720,7 @@ Fabrics.prototype.updateFabricsSvcs = function updateFabricsSvcs(cb) {
     if (self.dockerSetup) {
         svcs.push('docker');
     }
-    vasync.forEachParallel({
+    vasync.forEachPipeline({
         inputs: svcs,
         func: function updateSvc(svc, next) {
             common.spawnRun({
